_posts/2019-03-26-solidity-optimizer-and-abiencoderv2-bug.md

```diff
@@ -1,14 +1,12 @@
 ---
 layout: post
 published: true
-title: Solidity Optimizer and ABIEncoderV2 Bug
+title: Solidity Optimizer and ABIEncoderV2 Bugs
```

Comment:
I think it was just a single bug.

Reply:
Yes one bug for ABIEncoderV2 but also two bugs in the Solidity Optimizer?

Reply:
Ah sorry!